dotcom-rendering/tsconfig.json

```diff
@@ -63,10 +63,11 @@
         // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
         "forceConsistentCasingInFileNames": true,
         "resolveJsonModule": true,
-        "preserveConstEnums": true
+        "preserveConstEnums": true,
         /* Experimental Options */
         // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
         // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+        "skipLibCheck": true, /* temprarily disable lib checks until guardian/libs type error is fixed */
```

Comment:
Is it worth waiting for [this PR](https://github.com/guardian/libs/pull/319) to merge so we never need to add this

Reply:
Merged!